### PR TITLE
[release-v1.8] Migrate to use hack/generate tool to generate images.yaml

### PR DESCRIPTION
**Pull request by pierDipi**
Similar to eventing core in PRs like https://github.com/openshift-knative/eventing/pull/163

Followups:
- Support static-images in hack/generate tool
- Port this PR to release-next

### CODE REVIEW

File: openshift/ci-operator/static-images/dispatcher/Dockerfile
Comment (pierDipi) [reply openshift/ci-operator/static-images/dispatcher/Dockerfile:1]: We introduce static-images on main only, and this branch was created out of 1.7 midstream

Comment (matzew) [reply openshift/ci-operator/static-images/dispatcher/Dockerfile:1]: +1

**Review by matzew — APPROVED** [review]
/lgtm

looks good, especially for keeping the `ubi` images (JDK rt and minimal for go)

/approve

/hold
(as others should review too)

**Review by creydr — APPROVED** [review]
/lgtm

**Pull request merged**
